report_company_details_translatable/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This module allows to translate the company details.
```

Comment:
```suggestion
This module allows to translate the company details to show in PDF reports.

In some countries where the dominant language is not based on Roman alphabet, it is often necessary to be able to print the company details according to the partner's language.  In case of Japan, typically Japanese for domestic transactions, and English for international ones, for example.  This module intends to cater to this requirement.
```